packngo.go

```diff
@@ -144,6 +144,7 @@ type Client struct {
 	SpotMarket             SpotMarketService
 	SpotMarketRequests     SpotMarketRequestService
 	Organizations          OrganizationService
+	TransferRequest        TransferRequestsService
```

Comment:
Maybe rather `TransferRequestService` (TranferRequest in singular) to keep it uniform with other Service type names.